src/codecs/encoding/transformer.rs

```diff
@@ -356,4 +357,16 @@ mod tests {
         "#});
         assert!(config.is_err())
     }
+
+    #[test]
+    fn deny_unknown_fields() {
+        // We're only checking this explicitly because of our custom deserializer arrangement to
+        // make it possible to throw the exclusivity error during deserialization, to ensure that we
+        // enforce this on the top-level `Transformer`g type even though it has to be applied at the
```

Comment:
Is that `g` char after `Transformer` , a typo ?

Reply:
Ah, nice catch. That was indeed a woopsie daisy.

Fixed in ad7006dba.